code/datums/uplink_item.dm

```diff
@@ -485,6 +485,15 @@ var/list/uplink_items = list()
 	discounted_cost = 0
 	jobs_with_discount = SCIENCE_POSITIONS
 
+/datum/uplink_item/device_tools/reportintercom
+	name = "NTT Report Falsifier"
+	desc = "An intercom stolen from Nanotrasen Command that allows a single fake Galactic Update to be sent. Single-Use only. Let the crew know about the upcoming disk inspection."
+	item = /obj/item/device/reportintercom
+	num_in_stock = 3
+	cost = 6
+	discounted_cost = 4
+	jobs_with_discount = list("Nuclear Operative")
```

Comment:
Not sure if this will work. Nuke ops is a role, not a job.

Reply:
![EBIN DISK INSPECTION](https://i.imgur.com/npYLLAM.png)

Appears to work when I test it out by spawning the Nuke Ops Boombox